Source/Renderer/TimerQuery.js

```diff
@@ -0,0 +1,64 @@
+define([
+        '../Core/WebGLConstants',
+        '../Core/oneTimeWarning'
+    ], function(
+        WebGLConstants,
+        oneTimeWarning) {
+    'use strict';
+
+    /**
+     * Represents a timer query.
+     *
```

Comment:
Extra whitespace and I think this is missing `@alias`.  For reference, check other doc or the [doc guide](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/DocumentationGuide).